test/sandbox.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Can we do `sanboxed-iframe` and `sandboxed-iframe-contents`? I think that is a good balance between overly specific and generic